### PR TITLE
Migration to Handlebars

### PR DESCRIPTION
**Pull request by benl95**
- Deleted Pug and the Pug view pages
- Installed Handlebars template engine. 
- Refactored code in index.js to adhere to Handlebars setup.
- Created .prettierrc file, configured it and installed prettier as a devDependecy.

### CODE REVIEW

**Comment by rijkvanzanten** [comment]
@benl95 Next time try undoing the commits, instead of removing code and commiting that as a new point, cleans up the history a bit

**Comment by rijkvanzanten** [comment]
~You should leave package-lock (or yarn.lock) commited for websites like these to ensure everyone is working with the same combination of versions~

Nevermind! Looked like it was deleted, but there are just a lot of deletions within the file

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
☝️

**Comment by benl95** [comment]
> ☝️

Removed commits regarding search function

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="css/styles.css">
+	<script src="https://kit.fontawesome.com/d2dbc0198b.js" crossorigin="anonymous"></script>
```
Comment (rijkvanzanten) [inline views/layouts/main.handlebars]: Lets just use individual SVGs instead of relying on a full icon library. We'll only have like 5 icons anyway

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

**Comment by rijkvanzanten** [comment]
@benl95 can you re-add the package-lock.json file?

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**